Src/hikingAlgorithm.java

```diff
@@ -0,0 +1,5 @@
+public class hikingAlgorithm{
```

Comment:
This file must have been accidentally included. Delete this file from the project.